test/DynamoCoreTests/Configuration/PreferenceSettingsTests.cs

```diff
@@ -62,14 +62,39 @@ public void TestSetIsBackgroundPreviewActive()
                 initalSetting.GetIsBackgroundPreviewActive("IsBackgroundPreviewActive"));
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void TestSetDisplayCBNLineNumber()
+        {
+            string tempPath = System.IO.Path.GetTempPath();
+            tempPath = Path.Combine(tempPath, "userPreference.xml");
+
+            // Force initial state
+            PreferenceSettings initalSetting = new PreferenceSettings();
```

Comment:
typo: `initialSetting`

Reply:
Oops, copied testing code, need to fix for all the spelling in this file